rpc/test/client_test.go

```diff
@@ -64,17 +65,18 @@ func TestJSONBroadcastTxSync(t *testing.T) {
 }
 
 func testBroadcastTxSync(t *testing.T, client rpc.HTTPClient) {
-	config.Set("block_size", 0)
-	defer config.Set("block_size", -1)
+	//	config.Set("block_size", 0)
+	//	defer config.Set("block_size", -1)
```

Comment:
Commented out because this was involved in the old (unsafe) way of performing this test, which caused race errors to the viper config... aka I modified this test routine but left the comment in there in-case. will remove though